### PR TITLE
Dynamic reply url

### PR DESCRIPTION
**Pull request by majikandy**
.

### CODE REVIEW

**Comment by TimboTambo** [comment]
A test or two would be quite nice...

**Comment by majikandy** [comment]
> A test or two would be quite nice...

Just around the url split? Makes sense

File: medicines/pars-upload/src/auth/authPopup.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { UserAgentApplication } from 'msal'
 import { msalConfig, loginRequest } from './authConfig'
 
 export async function getAccount() {
+  msalConfig.auth.redirectUri = getCurrentHost(window.location.href)
```
Comment (sgrowe) [inline medicines/pars-upload/src/auth/authPopup.js:5]: Would it make sense to put this bit in `authConfig.js` instead?

File: medicines/pars-upload/src/auth/authPopup.js [hunk 2]
```diff
@@ -22,7 +22,13 @@ export async function getAccount() {
   }
 }
 
+export function getCurrentHost(currentFullUrl) {
+  var uriComponents = currentFullUrl.split('/')
+  return uriComponents[0] + '//' + uriComponents[2]
+}
+
 export function signIn() {
+  msalConfig.auth.redirectUri = getCurrentHost(window.location.href)
```
Comment (sgrowe) [inline medicines/pars-upload/src/auth/authPopup.js:31]: ^ Same as above 

> Would it make sense to put this bit in `authConfig.js` instead?

File: medicines/pars-upload/src/auth/authPopup.js (hunk 1 above)
Comment (majikandy) [reply medicines/pars-upload/src/auth/authPopup.js:5]: I had it there initially  but it couldn't access `window`

**Pull request merged**